CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
 if(ENABLE_MASON)
   # versions in use
-  set(MASON_BOOST_VERSION "1.65.1")
+  set(MASON_BOOST_VERSION "1.73.0")
```

Comment:
Upgrade Mason Boost to 1.73 as this is the first version >= 1.70 Mason provides that has all the libraries OSRM needs.